fpgalib/dac.py

```diff
@@ -85,41 +85,43 @@
 
 # TODO: Integrate this with DAC class
 # TODO: Have dac_calibration_server not DACify SRAM data
-def dacify(wave_a, wave_b, trigger_idx=(16, 17)):
-    """Construct SRAM sequence for a list of waveforms.
+
+
+SRAM_MAX = 0x1FFF  # 8190 = 2**13 - 1
+
+
+def dacify(wave_a, wave_b, trigger_idx=None):
```

Comment:
Could you change the default triggers in fpgatester to compensate for default value removal?